proxy.go

```diff
@@ -83,35 +84,58 @@ func NewProxyFromConfig(conf ProxyConfig) (p Proxy, err error) {
 	p.ConsulForwardService = conf.ConsulForwardServiceName
 	p.ConsulTraceService = conf.ConsulTraceServiceName
 
-	if p.ConsulForwardService != "" {
+	if p.ConsulForwardService != "" || conf.ForwardAddress != "" {
 		p.AcceptingForwards = true
 	}
-	if p.ConsulTraceService != "" {
+	if p.ConsulTraceService != "" || conf.TraceAddress != "" {
 		p.AcceptingTraces = true
 	}
 
+	// We need a convenient way to know if we're even using Consul later
+	if p.ConsulForwardService != "" || p.ConsulTraceService != "" {
+		p.usingConsul = true
+	}
+
+	p.ForwardDestinations = consistent.New()
+	p.TraceDestinations = consistent.New()
+
+	// We got a static forward address, stick it in the destination!
+	if p.ConsulForwardService == "" && conf.ForwardAddress != "" {
+		p.ForwardDestinations.Add(conf.ForwardAddress)
+	}
+	if p.ConsulTraceService == "" && conf.TraceAddress != "" {
+		p.TraceDestinations.Add(conf.TraceAddress)
+	}
+
 	if !p.AcceptingForwards && !p.AcceptingTraces {
-		err = errors.New("Refusing to start with no Consul service names in config.")
+		err = errors.New("Refusing to start with no Consul service names or static addresses in config.")
```

Comment:
nit: error strings should start with a lowercase letter, but also, if we're defining this on a static string, we should make it a package-level named error